service/src/main/java/bio/terra/axonserver/utils/GcpUtils.java

```diff
@@ -0,0 +1,19 @@
+package bio.terra.axonserver.utils;
+
+import bio.terra.axonserver.service.iam.AuthenticatedUserRequest;
+import bio.terra.stairway.Step;
+import com.google.auth.oauth2.AccessToken;
+import com.google.auth.oauth2.GoogleCredentials;
+
+/** Utilities for interacting with Google Cloud APIs within {@link Step}s. */
+public class GcpUtils {
+
+  private GcpUtils() {}
+
+  public static GoogleCredentials getGoogleCredentialsFromUserRequest(
+      AuthenticatedUserRequest userRequest) {
```

Comment:
If you are going to make your own AuthenticatedUserRequest, you could make this a method in it.
My bias is still to use the SamUser and BearerToken.